lib/src/mt_ptp.c

```diff
@@ -29,6 +29,17 @@ static char* ptp_mode_strs[MT_PTP_MAX_MODE] = {
     "l4",
 };
 
+#define KP 0.7
```

Comment:
struct mt_ptp_impl already has KP and KI, can we reuse them?
```
struct mt_ptp_impl {
  /* pi controller */
  bool use_pi;        /* use pi controller */
  double kp;          /* proportional gain */
  double ki;          /* integral gain */
  double integral;    /* integral value */
  int64_t prev_error; /* previous error (correct_delta) */
}
```

Reply:
> struct mt_ptp_impl already has KP and KI, can we reuse them?
> 
> ```
> struct mt_ptp_impl {
>   /* pi controller */
>   bool use_pi;        /* use pi controller */
>   double kp;          /* proportional gain */
>   double ki;          /* integral gain */
>   double integral;    /* integral value */
>   int64_t prev_error; /* previous error (correct_delta) */
> }
> ```

The duplicated constants KP&KI removed.